.chloggen/pprofile-starttime.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pdata/pprofile
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove field Profile.StartTime
```

Comment:
I don't think a changelog entry is required here. This has no user-facing changes, since this this keeps using the same input and output proto fields.

Reply:
dropped the changelog with https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/41074/commits/be2efc9f7751a29b923169d8951a39aff7187d8f.

Reply:
An approver of contrib will be able to do that.